python/tvm/contrib/sdaccel.py

```diff
@@ -35,8 +84,13 @@ def compile_vhls(code, kernel):
                             "sw_emu" if os.environ.get("XCL_EMULATION_MODE") else "hw")
     advanced_params = ["--xp", "param:compiler.preserveHlsOutput=1",
                        "--xp", "param:compiler.generateExtraRunData=true"]
-    platform = os.environ.get("XCL_PLATFORM",
-                              os.environ.get("AWS_PLATFORM", "xilinx:kcu1500:dynamic"))
+    platform = os.environ.get("XCL_PLATFORM", os.environ.get("AWS_PLATFORM"))
+
+    if platform is None:
```

Comment:
Suggest showing a warning like "SDAccel not found, running on GPU" in case if a user really wants to run with SDAccel but just forgets to set up the environment.